qt/aqt/stats.py

```diff
@@ -69,6 +80,11 @@ def closeWithCallback(self, callback: Callable[[], None]) -> None:
         self.reject()
         callback()
 
+    def on_deck_changed(self, deck_id: int) -> None:
+        set_current_deck(parent=self.mw, deck_id=DeckId(deck_id)).success(
+            lambda _: self.refresh()
+        ).run_in_background(initiator=self)
```

Comment:
Is this the right way to change the current deck?

Reply:
Yep, with some minor tweaks I've applied below.